packages/react-composites/tests/browser/call/hermetic/ErrorBar.test.ts

```diff
@@ -1,53 +1,50 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { buildUrlWithMockAdapter, test } from './fixture';
+import { buildUrlWithMockAdapterNext, defaultMockCallAdapterState, test } from './fixture';
 import { expect, Page } from '@playwright/test';
 import { dataUiId, stableScreenshot, waitForSelector } from '../../common/utils';
 import { IDS } from '../../common/constants';
 
 test.describe('Error bar tests', async () => {
   test('Failure to start video should be shown on error bar', async ({ page, serverUrl }) => {
-    await page.goto(
-      buildUrlWithMockAdapter(serverUrl, {
-        latestErrors: {
-          'Call.startVideo': {
-            timestamp: new Date(),
-            name: 'Failure to start video',
-            message: 'Could not start video',
-            target: 'Call.startVideo',
-            innerError: new Error('Inner error of failure to start video')
-          }
-        }
-      })
-    );
+    const initialState = defaultMockCallAdapterState();
+    initialState.latestErrors = {
+      'Call.startVideo': {
+        timestamp: new Date(),
+        name: 'Failure to start video',
+        message: 'Could not start video',
+        target: 'Call.startVideo',
+        innerError: new Error('Inner error of failure to start video')
+      }
+    };
+    await page.goto(buildUrlWithMockAdapterNext(serverUrl, initialState));
+
     await waitForSelector(page, dataUiId(IDS.videoGallery));
     expect(await stableScreenshot(page, { dismissTooltips: true })).toMatchSnapshot(
       'failure-to-start-video-on-error-bar.png'
     );
   });
 
   test('Multiple errors should be shown on error bar', async ({ page, serverUrl }) => {
-    await page.goto(
-      buildUrlWithMockAdapter(serverUrl, {
-        latestErrors: {
-          'Call.unmute': {
-            timestamp: new Date(),
-            name: 'Failure to unmute',
-            message: 'Could not unmute',
-            target: 'Call.unmute',
-            innerError: new Error('Inner error of failure to unmute')
-          },
-          'Call.stopVideo': {
-            timestamp: new Date(),
-            name: 'Failure to stop video',
-            message: 'Could not stop video',
-            target: 'Call.stopVideo',
-            innerError: new Error('Inner error of failure to stop video')
-          }
-        }
-      })
-    );
+    const initialState = defaultMockCallAdapterState();
+    initialState.latestErrors = {
+      'Call.unmute': {
+        timestamp: new Date(),
```

Comment:
Do these time stamps need to be real times? or would just giving them a dummy value be better?

Reply:
I guess not necessary, but I'm avoiding any semantic changes in this PR.